### PR TITLE
Typo in overview section of docs.

### LINKED ISSUE #210: Wrong JavaScript `pow` output in "Common Features' JS Output" table
In the "Common Features' JS Output" table, the `**` operator JavaScript output example is wrong:
``` mdx
Float Exponentiation            | `2.0 ** 3.0`                         | `Math.pow(3, 4)`
```
Expected output should be:
``` mdx
Float Exponentiation            | `2.0 ** 3.0`                         | `Math.pow(2.0, 3.0)`
```

### Affected files:
pages/docs/manual/latest/overview.mdx:223
pages/docs/manual/v8.0.0/overview.mdx:225

### PR DESCRIPTION
**Pull request by bdunn313**
Fixes #210

### CODE REVIEW

File: pages/docs/manual/latest/overview.mdx [hunk 1]
```diff
@@ -220,7 +220,7 @@ Integer Addition                | `23 + 1`                             | `23 + 1
 Float Addition                  | `23.0 +. 1.0`                        | `23.0 + 1.0`
 Integer Division/Multiplication | `2 / 23 * 1`                         | `2 / 23 * 1`
 Float Division/Multiplication   | `2.0 /. 23.0 *. 1.0`                 | `2.0 / 23.0 * 1.0`
-Float Exponentiation            | `2.0 ** 3.0`                         | `Math.pow(3, 4)`
+Float Exponentiation            | `2.0 ** 3.0`                         | `Math.pow(3.0, 4.0)`
 String Concatenation            | `"Hello " ++ "World"`                | `"Hello " + "World"`
```
Comment (bobzhang) [inline pages/docs/manual/latest/overview.mdx:224]: This actually needs to be changed into `Math.pow(2.0,3.0)`

File: pages/docs/manual/v8.0.0/overview.mdx [hunk 2]
```diff
@@ -222,7 +222,7 @@ Integer Addition                | `23 + 1`                             | `23 + 1
 Float Addition                  | `23.0 +. 1.0`                        | `23.0 + 1.0`
 Integer Division/Multiplication | `2 / 23 * 1`                         | `2 / 23 * 1`
 Float Division/Multiplication   | `2.0 /. 23.0 *. 1.0`                 | `2.0 / 23.0 * 1.0`
-Float Exponentiation            | `2.0 ** 3.0`                         | `Math.pow(3, 4)`
+Float Exponentiation            | `2.0 ** 3.0`                         | `Math.pow(3.0, 4.0)`
```
Comment (bobzhang) [inline pages/docs/manual/v8.0.0/overview.mdx:225]: same as above

File: pages/docs/manual/latest/overview.mdx (hunk 1 above)
Comment (bdunn313) [reply pages/docs/manual/latest/overview.mdx:224]: 🤦‍♂️fixed! Thanks.

File: pages/docs/manual/v8.0.0/overview.mdx (hunk 2 above)
Comment (bdunn313) [reply pages/docs/manual/v8.0.0/overview.mdx]: 🤦‍♂️fixed. Thanks!

**Pull request merged**